.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Fairlearn version [e.g. 0.4.4]
```

Comment:
As we discovered with Abe, we should explicitly ask if they installed via pip or by checking out from GitHub.

Reply:
I added a pip freeze instruction which would tell us how it's installed. Additionally, I've also asked for the Fairlearn version just to be sure.